<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Hu et al, 2023 (The Journal of Physical Chemistry B): Correlated protein modules — hu2023_K562 • scpdata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Hu et al, 2023 (The Journal of Physical Chemistry B): Correlated protein modules — hu2023_K562"><meta property="og:description" content="They demonstrate the correlations between the levels of pairs of proteins
in single-cell proteomics (SCP) at steady state. In measuring pairwise
correlations among 1000 proteins in a population of K562 cells and oocytes,
they observed many correlated protein modules (CPMs) that are functionally
involved in certain biological functions. Certain CPMs are specific to a
particular cell type, some common to different cell types. Additionally,
compared to single-cell transcriptomics and bulk proteomics,
protein correlations are functionally and experimentally more significant
in SCP than those corresponding mRNAs."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">


    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.17.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/contribution_guidelines.html">Contribution guidelines</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"></ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Hu et al, 2023 (The Journal of Physical Chemistry B): Correlated protein modules</h1>

    <div class="hidden name"><code>hu2023_K562.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>They demonstrate the correlations between the levels of pairs of proteins
in single-cell proteomics (SCP) at steady state. In measuring pairwise
correlations among 1000 proteins in a population of K562 cells and oocytes,
they observed many correlated protein modules (CPMs) that are functionally
involved in certain biological functions. Certain CPMs are specific to a
particular cell type, some common to different cell types. Additionally,
compared to single-cell transcriptomics and bulk proteomics,
protein correlations are functionally and experimentally more significant
in SCP than those corresponding mRNAs.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="va">hu2023_K562</span></span></code></pre></div>
    </div>

    <div id="format">
    <h2>Format</h2>
    <p>Two SingleCellExperiment objects:</p><ul><li><p><code>proteins_K562</code>: protein data containing quantitative data for 1249
proteins and 69 single-cells with zero imputation.</p></li>
<li><p><code>proteins_oocyte</code>: protein data containing quantitative data for 3422
proteins and 137 single-cells with zero imputation.</p></li>
</ul><p>The <code>colData(hu2023_oocyte())</code> contains cell type annotation.
The <code>colData(hu2023_K562())</code> contains cell type annotation.</p>
    </div>
    <div id="source">
    <h2>Source</h2>
    <p>The oocyte data were downloaded from the <a href="https://biopic-my.sharepoint.cn/:x:/g/personal/humo_biopic_pku_edu_cn/EfX4CHedVopLuSx2OJNj6LABdESGNdKz4Eh8Zawvd-fNNQ?e=E5m09k&amp;xsdata=MDV8MDJ8ZW5lcy5heWFyQHVjbG91dmFpbi5iZXxjYjY2M2MwYzNjMDY0YjZhNjc1NTA4ZGM4YzMzNjc1YXw3YWIwOTBkNGZhMmU0ZWNmYmM3YzQxMjdiNGQ1ODJlY3wxfDB8NjM4NTM5Mzk5NjI1Mzg1NDQ3fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazFoYVd3aUxDSlhWQ0k2TW4wPXwwfHx8&amp;sdata=Zmt4YnZFZFViTitJRkdTc0FTK2thMjdTT0EzV2JJeS83WlZmV3R6SzdvRT0%3d" class="external-link">Shared File</a>
The K563 cells protein data downloaded from the GitHub
(https://github.com/dionezhang/CPM/blob/master/ProteinAbundance.Rdata)
The raw data and the quantification data can also be found in the
MassIVE repository <code>MSV000089625</code>: ftp://MSV000089625@massive.ucsd.edu/.</p>
    </div>
    <div id="acquisition-protocol">
    <h2>Acquisition protocol</h2>



<p>The data were acquired using the following setup. More information
can be found in the source article (see <code>References</code>).</p><ul><li><p><strong>Cell isolation</strong>: K562 cells were re-suspended and washed in cold PBS.
Single cells/10 cells were sorted into 96-well plates using a FACSAria
instrument. Oocyte-cumulus complexes from C57/6J mice were collected
after PMSG and HCG injections, with hyaluronidase used to remove cumulus
cells. All samples stored at -80 degrees Celsius.</p></li>
<li><p><strong>Sample preparation</strong> Cells were digested with trypsin at 37 degrees
Celsius for 3 hours. For label-free proteomics, digestion was terminated
by adding 0.43% TFA and 1% ACN in water, followed by drying in a
concentrator. Peptides were resuspended in 0.1% TFA and 1% ACN, and
then transferred to sample tubes for LC-MS/MS analysis.</p></li>
<li><p><strong>Separation</strong>: 4 microliters of peptide digests were injected into a
high-performance chromatography column (IonOpticks) and separated at a
flow rate of 100 nL/min using a nanoflow liquid chromatography system.
The effective gradient was 70 mins, allowing 16 cells per day.</p></li>
<li><p><strong>Ionization</strong>: Peptides were analyzed using an Orbitrap Eclipse mass
spectrometer with a FAIMS Pro interface. FAIMS compensation voltages of
-55 and -70 V were applied, with a 1-second cycle time for both voltages.</p></li>
<li><p><strong>Mass spectrometry</strong>: MS spectra were acquired with the Orbitrap
analyzer, while MS/MS spectra were acquired with a linear ion trap
analyzer. The maximum ion injection time for MS/MS was 200 ms.</p></li>
<li><p><strong>Data analysis</strong>: MS raw files were searched against the UniProt
human protein database and an in-house contamination database
using Proteome Discoverer(2.4). Label-free quantification was based on
peak intensity with the match-between-runs (MBR) feature enabled.</p></li>
</ul></div>
    <div id="data-collection">
    <h2>Data collection</h2>



<p>The oocyte protein data shared by the author and it is accessible from the
<a href="https://biopic-my.sharepoint.cn/:x:/g/personal/humo_biopic_pku_edu_cn/EfX4CHedVopLuSx2OJNj6LABdESGNdKz4Eh8Zawvd-fNNQ?e=E5m09k&amp;xsdata=MDV8MDJ8ZW5lcy5heWFyQHVjbG91dmFpbi5iZXxjYjY2M2MwYzNjMDY0YjZhNjc1NTA4ZGM4YzMzNjc1YXw3YWIwOTBkNGZhMmU0ZWNmYmM3YzQxMjdiNGQ1ODJlY3wxfDB8NjM4NTM5Mzk5NjI1Mzg1NDQ3fFVua25vd258VFdGcGJHWnNiM2Q4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazFoYVd3aUxDSlhWQ0k2TW4wPXwwfHx8&amp;sdata=Zmt4YnZFZFViTitJRkdTc0FTK2thMjdTT0EzV2JJeS83WlZmV3R6SzdvRT0%3d" class="external-link">Shared File</a>
The K563 protein data is accessible from the
GitHub (https://github.com/dionezhang/CPM/blob/master/ProteinAbundance.Rdata).</p><ul><li><p><code>DataMatrix-oocyte-20240614.csv</code>: normalized imputed protein matrix</p></li>
<li><p><code>ProteinAbundance.Rdata</code>: protein matrices (normalized, log transformed)</p></li>
</ul><p>We initialized an empty QFeatures object and added the corresponding
protein assays as SingleCellExperiment objects.</p>
<p>The oocyte protein data were exported from the shared link as
(<code>DataMatrix-oocyte-20240614.csv</code>). The data were formatted to a
SingleCellExperiment object and the SampleType information were added
as only metadata, and stored in the <code>colData</code>. The object is then added
to the <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object.</p>
<p>The 562 cells protein data were downloaded from the GitHub link and loaded
to the memory. The <code>Norm</code> object were formatted to a SingleCellExperiment
object and the SampleType information were added as only metadata, and
stored in the <code>colData</code>. The object is then added to the <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object.</p>
    </div>
    <div id="references">
    <h2>References</h2>

<ul><li><p>Hu, M., Zhang, Y., Yuan, Y., Ma, W., Zheng, Y., Gu, Q., &amp; Xie, X. S. 2023.
“Correlated protein modules revealing functional coordination of
interacting proteins are detected by single-cell proteomics.”. The Journal
of Physical Chemistry B, (<a href="https://doi.org/10.1021/acs.jpcb.3c00014" class="external-link">link to article</a>).</p></li>
</ul></div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># \donttest{</span></span></span>
<span class="r-in"><span><span class="fu">hu2023_oocyte</span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> require(“SingleCellExperiment”)</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> class: SingleCellExperiment </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> dim: 3422 137 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> metadata(0):</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> assays(1): ''</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rownames(3422): P16125 Q8K3V4 ... Q91VU8 Q08274</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rowData names(0):</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colnames(137): h1 h3 ... z83 z86</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colData names(1): SampleType</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> reducedDimNames(0):</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> mainExpName: NULL</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> altExpNames(0):</span>
<span class="r-in"><span><span class="fu">hu2023_K562</span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> class: SingleCellExperiment </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> dim: 1249 69 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> metadata(0):</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> assays(1): ''</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rownames(1249): P49327 P08238 ... Q969Q0 O15400</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> rowData names(2): Description protein</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colnames(69): F1 F2 ... F94 F96</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> colData names(1): SampleType</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> reducedDimNames(0):</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> mainExpName: NULL</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> altExpNames(0):</span>
<span class="r-in"><span><span class="co"># }</span></span></span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.3.9000.</p>
</div>

      </footer></div>






  </body></html>

