<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Khan et al, 2023 (biorRxiv): Epithelial–Mesenchymal Transition — khan2023 • scpdata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Khan et al, 2023 (biorRxiv): Epithelial–Mesenchymal Transition — khan2023"><meta property="og:description" content="Single-cell samples were prepared using the nPOP sample
preparation method.  Proteomics data were acquired using the
SCoPE2 protocol on a Thermo Scientific Q-Exactive mass
spectrometer. The dataset contains quantitative information on 421
MCF-10A single cells undergoing epithelial–mesenchymal transition
(EMT) triggered by TGF beta. The data are available at the PSM,
and protein levels. The paper investigates the dynamics of
correlation modules at the protein level."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">


    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.15.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/contribution_guidelines.html">Contribution guidelines</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"></ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Khan et al, 2023 (biorRxiv): Epithelial–Mesenchymal Transition</h1>

    <div class="hidden name"><code>khan2023.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Single-cell samples were prepared using the nPOP sample
preparation method.  Proteomics data were acquired using the
SCoPE2 protocol on a Thermo Scientific Q-Exactive mass
spectrometer. The dataset contains quantitative information on 421
MCF-10A single cells undergoing epithelial–mesenchymal transition
(EMT) triggered by TGF beta. The data are available at the PSM,
and protein levels. The paper investigates the dynamics of
correlation modules at the protein level.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="va">khan2023</span></span></code></pre></div>
    </div>

    <div id="format">
    <h2>Format</h2>
    <p>A <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object with 47 assays, each assay being a
SingleCellExperiment object:</p><ul><li><p>Assay 1-44: PSM data acquired with a TMTPro 16plex protocol, hence
those assays contain 16 columns. Columns hold quantitative information
from single-cell channels, carrier channels, reference channels,
empty (negative control) channels and unused channels.</p></li>
<li><p><code>peptides</code>: peptide data containing quantitative data for 10055
peptides and 421 single-cells.</p></li>
<li><p><code>proteins_imputed</code>: protein data containing quantitative data for 4571
proteins and 420 single-cells with k-nearest neighbors (KNN) imputation.</p></li>
<li><p><code>proteins_unimputed</code>: protein data containing quantitative data for 4571
proteins and 420 single-cells without imputation.</p></li>
</ul><p>The <code>colData(khan2023())</code> contains cell type and batch annotations that
are common to all assays. The description of the <code>rowData</code> fields for the
PSM data can be found in the
<a href="https://cox-labs.github.io/coxdocs/output_tables.html" class="external-link"><code>MaxQuant</code> documentation</a>.</p>
    </div>
    <div id="source">
    <h2>Source</h2>
    <p>The data were downloaded from the
<a href="https://scp.slavovlab.net/Khan_et_al_2023" class="external-link">Slavov Lab</a> website via a
shared Google Drive
<a href="https://drive.google.com/drive/folders/1zCsRKWNQuAz5msxx0DfjDrIe6pUjqQmj" class="external-link">folder</a>.
The raw data and the quantification data can also be found in the
MassIVE repository <code>MSV000092872</code>:
ftp://MSV000092872@massive.ucsd.edu/.</p>
    </div>
    <div id="acquisition-protocol">
    <h2>Acquisition protocol</h2>



<p>The data were acquired using the following setup. More information
can be found in the source article (see <code>References</code>).</p><ul><li><p><strong>Cell isolation</strong>: CellenONE cell sorting.</p></li>
<li><p><strong>Sample preparation</strong> performed using the SCoPE2 protocol. nPOP
cell lysis (DMSO) + trypsin digestion + TMTPro 16plex protocol.</p></li>
<li><p><strong>Separation</strong>: online nLC (DionexUltiMate 3000 UHPLC with a
25cm x 75um IonOpticks Odyssey Series column (ODY3-25075C18); 200nL/min).</p></li>
<li><p><strong>Ionization</strong>: ESI (1,700 V).</p></li>
<li><p><strong>Mass spectrometry</strong>: Thermo Scientific Q-Exactive (MS1
resolution = 70,000; MS1 accumulation time = 300ms; MS2
resolution = 70,000).</p></li>
<li><p><strong>Data analysis</strong>: MaxQuant(2.4.13.0) + DART-ID.</p></li>
</ul></div>
    <div id="data-collection">
    <h2>Data collection</h2>



<p>The PSM data were collected from a shared Google Drive folder that
is accessible from the SlavovLab website (see <code>Source</code> section).
The folder ('/002-singleCellDataGeneration') contains the following
files of interest:</p><ul><li><p><code>ev_updated_NS.DIA.txt</code>: the MaxQuant/DART-ID output file</p></li>
<li><p><code>annotation.csv</code>: sample annotation</p></li>
<li><p><code>batch.csv</code>: batch annotation</p></li>
</ul><p>We combined the sample annotation and the batch annotation in
a single table. We also formatted the quantification table so that
columns match with those of the annotation and filter only for
single-cell runs. Both table are then combined in a single
<a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object using the <code><a href="https://UCLouvain-CBIO.github.io/scp/reference/readSCP.html" class="external-link">scp::readSCP()</a></code> function.</p>
<p>The peptide data were taken from the same google drive folder
(<code>EpiToMesen.TGFB.nPoP_trial1_pepByCellMatrix_NSThreshDART_medIntCrNorm.txt</code>).
The data were formatted to a SingleCellExperiment object and the sample
metadata were matched to the column names (mapping is retrieved
after running the SCoPE2 R script, <code>EMTTGFB_singleCellProcessing.R</code>) and
stored in the <code>colData</code>. The object is then added to the <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object
and the rows of the PSM data are linked to the rows of the peptide data
based on the peptide sequence information through an <code>AssayLink</code> object.</p>
<p>The imputed protein data were taken from the same google drive folder
(<code>EpiToMesen.TGFB.nPoP_trial1_1PercDartFDRTMTBulkDIA.WallE_imputed.txt</code>).
The data were formatted to a SingleCellExperiment object and the sample
metadata were matched to the column names (mapping is retrieved
after running the SCoPE2 R script, <code>EMTTGFB_singleCellProcessing.R</code>) and
stored in the <code>colData</code>. The object is then added to the <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object
and the rows of the peptide data are linked to the rows of the protein data
based on the protein sequence information through an <code>AssayLink</code> object.</p>
<p>The unimputed protein data were taken from the same google drive folder
(<code>EpiToMesen.TGFB.nPoP_trial1_1PercDartFDRTMTBulkDIA.WallE_unimputed.txt</code>).
The data were formatted and added exactly as imputed data.</p>
    </div>
    <div id="references">
    <h2>References</h2>
    <p>Saad Khan, Rachel Conover, Anand R. Asthagiri, Nikolai Slavov. 2023.
"Dynamics of single-cell protein covariation during epithelial–mesenchymal
transition." Journal of Proteome Research.
(<a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00277" class="external-link">link to article</a>).</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># \donttest{</span></span></span>
<span class="r-in"><span><span class="fu">khan2023</span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> An instance of class QFeatures containing 47 set(s):</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [1] eSK233: SingleCellExperiment with 5951 rows and 16 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [2] eSK234: SingleCellExperiment with 6375 rows and 16 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [3] eSK235: SingleCellExperiment with 6261 rows and 16 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [45] peptides: SingleCellExperiment with 10055 rows and 421 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [46] proteins_imputed: SingleCellExperiment with 4096 rows and 421 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [47] proteins_unimputed: SingleCellExperiment with 4096 rows and 421 columns </span>
<span class="r-in"><span><span class="co"># }</span></span></span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.9000.</p>
</div>

      </footer></div>






  </body></html>

