<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Leduc et al. 2022 - plexDIA (biorRxiv): melanoma cells — leduc2022_plexDIA • scpdata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Leduc et al. 2022 - plexDIA (biorRxiv): melanoma cells — leduc2022_plexDIA"><meta property="og:description" content="Single cell proteomics data acquired by the Slavov Lab. This is
the dataset associated to the fourth version of the preprint (and
the Genome Biology publication). It contains quantitative
information of melanoma cells at precursor, peptide and protein level.
This version of the data was acquired using the plexDIA MS
acquisition protocol."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.9.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/contribution_guidelines.html">Contribution guidelines</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"></ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Leduc et al. 2022 - plexDIA (biorRxiv): melanoma cells</h1>
    
    <div class="hidden name"><code>leduc2022_plexDIA.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Single cell proteomics data acquired by the Slavov Lab. This is
the dataset associated to the fourth version of the preprint (and
the Genome Biology publication). It contains quantitative
information of melanoma cells at precursor, peptide and protein level.
This version of the data was acquired using the plexDIA MS
acquisition protocol.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="va">leduc2022_plexDIA</span></span></code></pre></div>
    </div>

    <div id="format">
    <h2>Format</h2>
    <p>A <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object with 48 assays, each assay being a
SingleCellExperiment object:</p><ul><li><p>Assay 1-45: precursor data acquired with a mTRAQ-3 protocol,
hence those assays contain 3 columns. Columns hold quantitative
information from single cells or negative control samples.</p></li>
<li><p><code>Ms1Extracted</code>: the DIA-NN MS1 extracted signal, it combines the
information from assays 1-45.</p></li>
<li><p><code>peptides</code>: peptide data containing quantitative data for 3,608
peptides and 104 single cells. The  data were filtered  to  1%
protein FDR.</p></li>
<li><p><code>proteins</code>: protein data containing quantitative data for 508
proteins and 105 single cells. Note that the peptide and protein
data provided by the authors differ by 3 samples. The precursor
data were aggregated to protein intensity using maxLFQ. The
protein data were further median normalized by column and by row,
log2 transformed, impute using KNN (k = 3), again median
normalized by column and by row, batch corrected using ComBat,
and median normalized by column and by row once more.</p></li>
</ul><p>The <code>colData(leduc2022_plexDIA())</code> contains cell type annotation and
batch annotation that are common to all assays. The description of
the <code>rowData</code> fields for the precursor data can be found in the
<a href="https://github.com/vdemichev/DiaNN#readme" class="external-link"><code>DIA-NN</code> documentation</a>.</p>
    </div>
    <div id="source">
    <h2>Source</h2>
    <p>The links to the data were found on the
<a href="https://scp.slavovlab.net/Leduc_et_al_2022" class="external-link">Slavov Lab website</a>.
The data were downloaded from the
<a href="https://drive.google.com/drive/folders/117ZUG5aFIJt0vrqIxpKXQJorNtekO-BV" class="external-link">Google drive folder 1</a> and
<a href="https://drive.google.com/drive/folders/12-H2a1mfSHZUGf8O50Cr0pPZ4zIDjTac" class="external-link">Google drive folder 2</a>.
The raw data and the quantification data can also be found in the
massIVE repository <code>MSV000089159</code>:
ftp://massive.ucsd.edu/MSV000089159.</p>
    </div>
    <div id="acquisition-protocol">
    <h2>Acquisition protocol</h2>
    


<p>The data were acquired using the following setup. More information
can be found in the source article (see <code>References</code>).</p><ul><li><p><strong>Cell isolation</strong>: CellenONE cell sorting.</p></li>
<li><p><strong>Sample preparation</strong> performed using the improved SCoPE2
protocol using the CellenONE liquid handling system. nPOP cell
lysis (DMSO) + trypsin digestion + mTRAQ-3
labeling and pooling.</p></li>
<li><p><strong>Separation</strong>: online nLC (DionexUltiMate 3000 UHPLC with a
25cm x 75um IonOpticks Aurora Series UHPLC column; 200nL/min).</p></li>
<li><p><strong>Ionization</strong>: ESI (1,800V).</p></li>
<li><p><strong>Mass spectrometry</strong>: Thermo Scientific Q-Exactive. The duty
cycle = 1 MS1 + 4 DIA MS2 windows (120 Th, 120 Th, 200 Th and
580 Th, spanning 378-1,402 m/z). Each MS1 and MS2 scan was
conducted at 70,000 resolving power, 3×10E6 AGC and 300ms
maximum injection time.</p></li>
<li><p><strong>Data analysis</strong>: DIA-NN.</p></li>
</ul></div>
    <div id="data-collection">
    <h2>Data collection</h2>
    


<p>The PSM data were collected from a shared Google Drive folder that
is accessible from the SlavovLab website (see <code>Source</code> section).
The folder contains the following files of interest:</p><ul><li><p><code>annotation_plexDIA.csv</code>: sample annotation</p></li>
<li><p><code>report_plexDIA_mel_nPOP.tsv</code>: the DIA-NN output file
with the precursor data</p></li>
<li><p><code>report.pr_matrix_channels_ms1_extracted.tsv</code>: the DIA-NN
output file with the combined precursor data</p></li>
<li><p><code>plexDIA_peptide.csv</code>: the processed data table containing the
<code>peptide</code> data</p></li>
<li><p><code>plexDIA_protein_imputed.csv</code>: the processed data table
containing the <code>protein</code> data</p></li>
</ul><p>We removed the failed runs as identified by the authors. We also
formated the annotation and precuror quantification tables to
facilitate matching between corresponding columns. Both annotation
and quantification tables are then combined in a single <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a>
object using <code><a href="https://UCLouvain-CBIO.github.io/scp/reference/readSCPfromDIANN.html" class="external-link">scp::readSCPfromDIANN()</a></code>.</p>
<p>The <code>plexDIA_peptide.csv</code> and <code>plexDIA_protein_imputed.csv</code> files
were loaded and formated as SingleCellExperiment objects. The
columns names were adapted to match those in the <code>QFeatures</code>
object. The <code>SingleCellExperiment</code> objects were then added to the
<a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object and the rows of the peptide data are linked to
the rows of the precursor data based on the peptide sequence or
the protein name through an <code>AssayLink</code> object.</p>
    </div>
    <div id="references">
    <h2>References</h2>
    <p>Andrew Leduc, Gray Huffman, and Nikolai Slavov. 2022. “Droplet
Sample Preparation for Single-Cell Proteomics Applied to the Cell
Cycle.” bioRxiv. <a href="https://doi.org/10.1101/2021.04.24.441211" class="external-link">Link to article</a></p>
<p>Andrew Leduc, Gray Huffman, Joshua Cantlon, Saad Khan, and Nikolai
Slavov. 2022. “Exploring Functional Protein Covariation across
Single Cells Using nPOP.” Genome Biology 23 (1): 261.
<a href="http://dx.doi.org/10.1186/s13059-022-02817-5" class="external-link">Link to article</a></p>
<p>Jason Derks, Andrew Leduc, Georg Wallmann, Gray Huffman, Matthew
Willetts, Saad Khan, Harrison Specht, Markus Ralser, Vadim
Demichev, and Nikolai Slavov. 2023. “Increasing the Throughput of
Sensitive Proteomics by plexDIA.” Nature Biotechnology 41 (1):
50–59. <a href="http://dx.doi.org/10.1038/s41587-022-01389-w" class="external-link">Link to article</a></p>
    </div>
    <div id="see-also">
    <h2>See also</h2>
    <div class="dont-index"><p><a href="leduc2022_pSCoPE.html">leduc2022_pSCoPE</a></p></div>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># \donttest{</span></span></span>
<span class="r-in"><span><span class="fu">leduc2022_plexDIA</span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> An instance of class QFeatures containing 48 assays:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [1] D..AL.AL.wAL_plexMel01.raw: SingleCellExperiment with 2495 rows and 3 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [2] D..AL.AL.wAL_plexMel03.raw: SingleCellExperiment with 1738 rows and 3 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [3] D..AL.AL.wAL_plexMel05.raw: SingleCellExperiment with 2299 rows and 3 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [46] Ms1Extracted: SingleCellExperiment with 4682 rows and 135 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [47] peptides: SingleCellExperiment with 3608 rows and 104 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [48] proteins: SingleCellExperiment with 508 rows and 105 columns </span>
<span class="r-in"><span><span class="co"># }</span></span></span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.9000.</p>
</div>

      </footer></div>

  


  

  </body></html>

