<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Specht et al. 2019 - SCoPE2 (biorRxiv): macrophages vs monocytes
(version 2) — specht2019v2 • scpdata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Specht et al. 2019 - SCoPE2 (biorRxiv): macrophages vs monocytes
(version 2) — specht2019v2"><meta property="og:description" content="Single cell proteomics data acquired by the Slavov Lab. This is
the version 2 of the data released in December 2019. It contains
quantitative information of macrophages and monocytes at PSM,
peptide and protein level."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.5.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"></ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Specht et al. 2019 - SCoPE2 (biorRxiv): macrophages vs monocytes
(version 2)</h1>
    
    <div class="hidden name"><code>specht2019v2.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Single cell proteomics data acquired by the Slavov Lab. This is
the version 2 of the data released in December 2019. It contains
quantitative information of macrophages and monocytes at PSM,
peptide and protein level.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="va">specht2019v2</span></span></code></pre></div>
    </div>

    <div id="format">
    <h2>Format</h2>
    <p>A <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object with 179 assays, each assay being a
SingleCellExperiment object:</p><ul><li><p>Assay 1-63: PSM data for SCoPE2 sets acquired with a TMT-11plex
protocol, hence those assays contain 11 columns. Columns
hold quantitative information from single-cell channels, carrier
channels, reference channels, empty (blank) channels and unused
channels.</p></li>
<li><p>Assay 64-177: PSM data for SCoPE2 sets acquired with a
TMT-16plex protocol, hence those assays contain 16 columns.
Columns hold quantitative information from single-cell channels,
carrier channels, reference channels, empty (blank) channels and
unused channels.</p></li>
<li><p><code>peptides</code>: peptide data containing quantitative data for 9208
peptides and 1018 single-cells.</p></li>
<li><p><code>proteins</code>: protein data containing quantitative data for 2772
proteins and 1018 single-cells.</p></li>
</ul><p>The <code>colData(specht2019v2())</code> contains cell type annotation and
batch annotation that are common to all assays. The description of
the <code>rowData</code> fields for the PSM data can be found in the
<a href="http://www.coxdocs.org/doku.php?id=maxquant:table:evidencetable" class="external-link"><code>MaxQuant</code> documentation</a>.</p>
    </div>
    <div id="source">
    <h2>Source</h2>
    <p>The data were downloaded from the
<a href="https://scope2.slavovlab.net/mass-spec/data" class="external-link">Slavov Lab</a> website via a
shared Google Drive
<a href="https://drive.google.com/drive/folders/1VzBfmNxziRYqayx3SP-cOe2gu129Obgx" class="external-link">folder</a>.
The raw data and the quantification data can also be found in the
massIVE repository <code>MSV000083945</code>:
ftp://massive.ucsd.edu/MSV000083945.</p>
    </div>
    <div id="acquisition-protocol">
    <h2>Acquisition protocol</h2>
    


<p>The data were acquired using the following setup. More information
can be found in the source article (see <code>References</code>).</p><ul><li><p><strong>Cell isolation</strong>: flow cytometry (BD FACSAria I).</p></li>
<li><p><strong>Sample preparation</strong> performed using the SCoPE2 protocol. mPOP
cell lysis + trypsin digestion + TMT-11plex or 16plex labelling
and pooling.</p></li>
<li><p><strong>Separation</strong>: online nLC (DionexUltiMate 3000 UHPLC with a
25cm x 75um IonOpticksAurora Series UHPLC column; 200nL/min).</p></li>
<li><p><strong>Ionization</strong>: ESI (2,200V).</p></li>
<li><p><strong>Mass spectrometry</strong>: Thermo Scientific Q-Exactive (MS1
resolution = 70,000; MS1 accumulation time = 300ms; MS2
resolution = 70,000).</p></li>
<li><p><strong>Data analysis</strong>: DART-ID + MaxQuant (1.6.2.3).</p></li>
</ul></div>
    <div id="data-collection">
    <h2>Data collection</h2>
    


<p>The PSM data were collected from a shared Google Drive folder that
is accessible from the SlavovLab website (see <code>Source</code> section).
The folder contains the following files of interest:</p><ul><li><p><code>ev_updated.txt</code>: the MaxQuant/DART-ID output file</p></li>
<li><p><code>annotation_fp60-97.csv</code>: sample annotation</p></li>
<li><p><code>batch_fp60-97.csv</code>: batch annotation</p></li>
</ul><p>We combined the the sample annotation and the batch annotation in
a single table. We also formated the quantification table so that
columns match with those of the annotation and filter only for
single-cell runs. Both table are then combined in a single
<a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object using the <a href="https://UCLouvain-CBIO.github.io/scp/reference/readSCP.html" class="external-link">scp::readSCP</a> function.</p>
<p>The peptide data were taken from the Slavov lab directly
(<code>Peptides-raw.csv</code>). It is provided as a spreadsheet. The data
were formated to a SingleCellExperiment object and the sample
metadata were matched to the column names (mapping is retrieved
after running the SCoPE2 R script) and stored in the <code>colData</code>.
The object is then added to the <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object (containing the
PSM assays) and the rows of the peptide data are linked to the
rows of the PSM data based on the peptide sequence information
through an <code>AssayLink</code> object.</p>
<p>The protein data (<code>Proteins-processed.csv</code>) is formated similarly
to the peptide data, and the rows of the proteins were mapped onto
the rows of the peptide data based on the protein sequence
information.</p>
    </div>
    <div id="references">
    <h2>References</h2>
    <p>Specht, Harrison, Edward Emmott, Aleksandra A.
Petelski, R. Gray Huffman, David H. Perlman, Marco Serra, Peter
Kharchenko, Antonius Koller, and Nikolai Slavov. 2019.
"Single-Cell Mass-Spectrometry Quantifies the Emergence of
Macrophage Heterogeneity." bioRxiv.
(<a href="https://doi.org/10.1101/665307" class="external-link">link to article</a>).</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="fu">specht2019v2</span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> An instance of class QFeatures containing 179 assays:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [1] 190222S_LCA9_X_FP94AA: SingleCellExperiment with 2823 rows and 11 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [2] 190222S_LCA9_X_FP94AB: SingleCellExperiment with 4297 rows and 11 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [3] 190222S_LCA9_X_FP94AC: SingleCellExperiment with 4956 rows and 11 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [177] 191110S_LCB7_X_APNOV16plex2_Set_9: SingleCellExperiment with 4626 rows and 16 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [178] peptides: SingleCellExperiment with 9208 rows and 1018 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [179] proteins: SingleCellExperiment with 2772 rows and 1018 columns </span>
<span class="r-in"><span></span></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.9000.</p>
</div>

      </footer></div>

  


  

  </body></html>

