<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Williams et al. 2020 (Anal. Chem.): MCF10A cell line — williams2020_lfq • scpdata</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Williams et al. 2020 (Anal. Chem.): MCF10A cell line — williams2020_lfq"><meta property="og:description" content="Single-cell label free proteomics data from a MCF10A cell line
culture. The data were acquired using a label-free quantification
protocole based on the nanoPOTS technology. The objective was to
test 2 elution gradients for single-cell applications and to
demonstrate successful use of the new nanoPOTS autosampler
presented in the article. The samples contain either no cells,
single cells, 3 cells, 10 cells  50 cells."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.3.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"></ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Williams et al. 2020 (Anal. Chem.): MCF10A cell line</h1>
    
    <div class="hidden name"><code>williams2020_lfq.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Single-cell label free proteomics data from a MCF10A cell line
culture. The data were acquired using a label-free quantification
protocole based on the nanoPOTS technology. The objective was to
test 2 elution gradients for single-cell applications and to
demonstrate successful use of the new nanoPOTS autosampler
presented in the article. The samples contain either no cells,
single cells, 3 cells, 10 cells  50 cells.</p>
    </div>

    <div class="ref-usage sourceCode"><pre class="sourceCode r"><code><span class="va">williams2020_lfq</span></code></pre></div>

    <div id="ref-sections">
    <div id="format">
    <h2>Format</h2>
    <p>A <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object with 9 assays, each assay being a
SingleCellExperiment object:</p><ul><li><p><code>peptides_[30 or 60]min_[intensity or LFQ]</code>: 3 assays
containing peptide intensities or LFQ normalized
quantifications (see <code>References</code>) for either a 30min or a 60 min
gradient.</p></li>
<li><p><code>proteins_[30 or 60]min_[intensity or iBAQ or LFQ]</code>: 6 assays
containing protein intensities, iBAQ normalized or LFQ normalized
quantifications (see <code>References</code>) for either a 30min or a 60 min
gradient.</p></li>
</ul><p>Sample annotation is stored in <code>colData(williams2020_lfq())</code>.</p>
    </div>
    <div id="source">
    <h2>Source</h2>
    <p>The PSM and protein data can be downloaded from the MASSIVE
repository MSV000085230.</p>
    </div>
    <div id="acquisition-protocol">
    <h2>Acquisition protocol</h2>
    


<p>The data were acquired using the following setup. More information
can be found in the source article (see <code>References</code>).</p><ul><li><p><strong>Sample isolation</strong>: cultured MCF10A cells were isolated using
flow-cytometry based cell sorting and deposit on nanoPOTS
microwells</p></li>
<li><p><strong>Sample preparation</strong>: cells are lysed using using a DDM+DTT
lysis buffer. Alkylation was then performed using an IAA solution.
Proteins are digested with Lys-C and trypsin followed by
acidification with FA. Sample droplets are then dried until
LC-MS/MS analysis.</p></li>
<li><p><strong>Liquid chromatography</strong>: peptides are loaded using the new
autosampler described in the paper. Samples are loaded using a
a homemade miniature syringe pump. The samples are then desalted
and concentrated through a SPE column (4cm x 100µm i.d. packed
with 5µm C18) with microflow LC pump. The peptides are then eluted
from a long LC column (60cm x 50 µm i.d. packed with 3µm C18)
coupled to a nanoflox LC pump at 150nL/mL with either a 30 min
or a 60 min gradient.</p></li>
<li><p><strong>Mass spectrometry</strong>: MS/MS was performed on an Orbitrap Fusion
Lumos Tribrid MS coupled to a 2kV ESI. MS1 setup: Orbitrap
analyzer at resolution 120.000, AGC target of 1E6, accumulation
of 246ms. MS2 setup: ion trap with CID at resolution 60.000, AGC
target of 2E4, accumulation of 120ms (50 cells) or 250ms (0-10
cells).</p></li>
<li><p><strong>Raw data processing</strong>: preprocessing using Maxquant v1.6.2.10
that use Andromeda search engine (with UniProtKB 2016-21-29),
MBR and LFQ normalization were enabled.</p></li>
</ul></div>
    <div id="data-collection">
    <h2>Data collection</h2>
    


<p>All data were collected from the MASSIVE repository (accession ID:
MSV000085230).</p>
<p>The peptide and protein data were extracted from the <code>Peptides_[...].txt</code>
or <code>ProteinGroups[...].txt</code> files, respectively, in the
<code>MCF10A_LC_[30 or 60]minutes</code> folders.</p>
<p>The tables were duplicated so that peptide intensisities, peptide
LFQ, protein intensities, protein LFQ and protein intensities are
contained in separate tables. Tables are then converted to
SingleCellExperiment objects. Sample annotations were infered
from the sample names and from the paper. All data is combined in
a <a href="https://rdrr.io/pkg/QFeatures/man/QFeatures-class.html" class="external-link">QFeatures</a> object. <a href="https://rdrr.io/pkg/QFeatures/man/AssayLinks.html" class="external-link">AssayLinks</a> were stored between peptide
assays and their corresponding proteins assays based on the
leading razor protein (hence only unique peptides are linked to
proteins).</p>
<p>The script to reproduce the <code>QFeatures</code> object is available at
<code>system.file("scripts", "make-data_williams2020_lfq.R", package = "scpdata")</code></p>
    </div>
    <div id="suggestion">
    <h2>Suggestion</h2>
    


<p>See <code><a href="https://rdrr.io/pkg/QFeatures/man/joinAssays.html" class="external-link">QFeatures::joinAssays</a></code> if you want to join the 30min and
60min assays in a single assay for an integrated analysis.</p>
    </div>
    <div id="references">
    <h2>References</h2>
    <p><strong>Source article</strong>: Williams, Sarah M., Andrey V. Liyu, Chia-Feng
Tsai, Ronald J. Moore, Daniel J. Orton, William B. Chrisler,
Matthew J. Gaffrey, et al. 2020. “Automated Coupling of
Nanodroplet Sample Preparation with Liquid Chromatography-Mass
Spectrometry for High-Throughput Single-Cell Proteomics.”
Analytical Chemistry 92 (15): 10588–96.
(<a href="http://dx.doi.org/10.1021/acs.analchem.0c01551" class="external-link">link to article</a>).</p>
<p><strong>LFQ normalization</strong>: Cox, Jürgen, Marco Y. Hein, Christian A. Luber,
Igor Paron, Nagarjuna Nagaraj, and Matthias Mann. 2014. “Accurate
Proteome-Wide Label-Free Quantification by Delayed Normalization
and Maximal Peptide Ratio Extraction, Termed MaxLFQ.” Molecular
&amp; Cellular Proteomics: MCP 13 (9): 2513–26.
(<a href="http://dx.doi.org/10.1074/mcp.M113.031591" class="external-link">link to article</a>).</p>
<p><strong>iBAQ normalization</strong>: Schwanhäusser, Björn, Dorothea Busse, Na
Li, Gunnar Dittmar, Johannes Schuchhardt, Jana Wolf, Wei Chen, and
Matthias Selbach. 2011. “Global Quantification of Mammalian Gene
Expression Control.” Nature 473 (7347): 337–42.
(<a href="http://dx.doi.org/10.1038/nature10098" class="external-link">link to article</a>).</p>
    </div>
    </div>

    <div id="examples">
    <h2>Examples</h2>
    <div class="ref-examples sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span class="fu">williams2020_lfq</span><span class="op">(</span><span class="op">)</span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> loading from cache</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> An instance of class QFeatures containing 9 assays:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [1] peptides_30min_intensity: SingleCellExperiment with 8643 rows and 26 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [2] peptides_30min_LFQ: SingleCellExperiment with 8643 rows and 26 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [3] peptides_60min_intensity: SingleCellExperiment with 6265 rows and 9 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  ...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [7] proteins_60min_intensity: SingleCellExperiment with 1600 rows and 9 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [8] proteins_60min_iBAQ: SingleCellExperiment with 1600 rows and 9 columns </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [9] proteins_60min_LFQ: SingleCellExperiment with 1600 rows and 1 columns </span>
<span class="r-in"></span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 1.9000.9000.9000.</p>
</div>

      </footer></div>

  


  

  </body></html>

