<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Contribution guidelines • scpdata</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Contribution guidelines">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.15.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/contribution_guidelines.html">Contribution guidelines</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Contribution guidelines</h1>
                        <h4 data-toc-skip class="author">Christophe
Vanderaa</h4>
                                    <h4 data-toc-skip class="author">Laurent
Gatto</h4>
                        
            <h4 data-toc-skip class="date">6 May 2025</h4>
      

      <div class="hidden name"><code>contribution_guidelines.Rmd</code></div>

    </div>

    
    
<p>Welcome to the <code>scpdata</code> package, and thank you for your
interest in contributing!</p>
<p>The <code>scpdata</code> data package is a repository of curated mass
spectrometry-based single-cell proteomics (SCP) datasets. The purpose of
<code>scpdata</code> is to provide users with streamlined access to
high-quality SCP data, alleviating the need for time-consuming data
wrangling. We currently provide data at the peptide-to-spectrum match
(PSM) level, the peptide level and/or the protein level. The package
also encompasses a large diversity of technologies, including DDA and
DIA, label-free and multiplexed experiments from various laboratories
such as the Slavov Lab, the Kelly Lab, and the Schoof Lab.</p>
<p>Contributions are very much welcome. We happily accept major
contributions such as adding a new dataset, as well as minor
contributions as fixing typos or improving current documentation.</p>
<p>To facilitate our collaboration, this vignette will guide you through
the process of adding a new dataset to the package. We will first get
you started with some basic guidelines on how to contribute using
GitHub. We’ll proceed with a description of the data structure and the
data pieces we expect. Next, we will provide an overview of the
package’s folder structure to help you navigate through the project.
Finally, we’ll explain the workflow you should follow to add your
dataset to the repository.</p>
<div class="section level2">
<h2 id="getting-started-with-github">Getting started with GitHub<a class="anchor" aria-label="anchor" href="#getting-started-with-github"></a>
</h2>
<ol style="list-style-type: decimal">
<li>Fork the <code>scpdata</code> GitHub repository (<a href="https://github.com/UCLouvain-CBIO/scpdata/fork" class="external-link">click
here</a>).</li>
<li>Clone the forked repo locally using <code>git</code>:</li>
</ol>
<pre><code>git clone git@github.com:YOUR_USER_NAME/scpdata</code></pre>
<ol start="3" style="list-style-type: decimal">
<li>Adapt the cloned repo as desired. Do not forget to regularly `git
commit`` your changes.</li>
<li>Once finished, send your improvements and/or new features as a <a href="https://github.com/UCLouvain-CBIO/scpdata/compare" class="external-link">pull
request</a>.</li>
</ol>
<p>If you have any questions or face any hurdles, do not hesitate to
open a <a href="https://github.com/UCLouvain-CBIO/scpdata/issues/new/choose" class="external-link">new
issue</a> and we’ll be happy to provide additional guidance.</p>
</div>
<div class="section level2">
<h2 id="what-do-we-expect">What do we expect?<a class="anchor" aria-label="anchor" href="#what-do-we-expect"></a>
</h2>
<div class="section level3">
<h3 id="qfeatures-object">
<code>QFeatures</code> object<a class="anchor" aria-label="anchor" href="#qfeatures-object"></a>
</h3>
<p>All datasets in <code>scpdata</code> are stored in a
<code>QFeatures</code> object (see <a href="https://uclouvain-cbio.github.io/scp/articles/QFeatures_nutshell.html" class="external-link">intro
vignette</a>). The object is created following the <a href="https://github.com/UCLouvain-CBIO/scp" class="external-link"><code>scp</code></a> data
framework, as described in <a href="https://uclouvain-cbio.github.io/scp-teaching/read_scp_data" class="external-link">this
short demo</a>.</p>
<div class="section level4">
<h4 id="feature-data">Feature data<a class="anchor" aria-label="anchor" href="#feature-data"></a>
</h4>
<p>We refer to feature data as the data generated by MS data
identification and quantification tools. Depending on the tool, features
may represent PSMs, peptides and/or proteins. For instance, MaxQuant
provides an <code>evidence.txt</code> file with PSM-level information, a
<code>peptides.txt</code> file with peptide-level information and
<code>proteinGroups.txt</code> with protein-level information. We
encourage adding as many of the three feature layers when contributing a
dataset to <code>scpdata</code>.</p>
<p>For each feature, the tools provide quantification data as well as
feature annotations. These two pieces of information should be separated
in a <code>SingleCellExperiment</code> object. Feature annotations are
stored in the <code>rowData</code> and the quantitative values are
stored in the <code>assay</code>.</p>
</div>
<div class="section level4">
<h4 id="sample-annotations">Sample annotations<a class="anchor" aria-label="anchor" href="#sample-annotations"></a>
</h4>
<p>Sample annotations contain information about each sample (single
cell) in the dataset. This information is generated by the experimenter
and should contain biological descriptors, such as the cell line or the
treatment applied, and technical descriptors, such as the day of
acquisition, the acquisition batch, the LC batch, etc. The sample
annotations are stored in the <code>colData</code> of the
<code>QFeatures</code> object.</p>
<p>If you want to contribute to <code>scpdata</code> with a dataset you
generated yourself, we suggest you read the last section of initial
recommendations for SCP experiments that provides a comprehensive
discussion about descriptors of interest you should collect:</p>
<blockquote>
<p>Gatto, Laurent, Ruedi Aebersold, Juergen Cox, Vadim Demichev, Jason
Derks, Edward Emmott, Alexander M. Franks, et al. 2023. “Initial
Recommendations for Performing, Benchmarking and Reporting Single-Cell
Proteomics Experiments.” Nature Methods 20 (3): 375–86.</p>
</blockquote>
</div>
</div>
<div class="section level3">
<h3 id="experiment-description">Experiment description<a class="anchor" aria-label="anchor" href="#experiment-description"></a>
</h3>
<p>We also require the collection of experimental data that describes
the dataset. This information is commonly retrieved from the publication
associated with the dataset and provides a scientific context to the
dataset. This information is used for building the dataset
documentation.</p>
</div>
<div class="section level3">
<h3 id="data-source-information">Data source information<a class="anchor" aria-label="anchor" href="#data-source-information"></a>
</h3>
<p>Finally, the <code>ExperimentHub</code> project, on which
<code>scpdata</code> relies, requires every dataset to thoroughly
provide a description of the data sources.</p>
</div>
</div>
<div class="section level2">
<h2 id="folder-structure">Folder structure<a class="anchor" aria-label="anchor" href="#folder-structure"></a>
</h2>
<p>We here provide an overview of the key folders and files relevant
when contributing a new dataset. The current files may provide a source
of inspiration when preparing a new dataset.</p>
<div class="section level3">
<h3 id="instscripts">inst/scripts/<a class="anchor" aria-label="anchor" href="#instscripts"></a>
</h3>
<p>The folder contains all R scripts used to generate the
<code>QFeatures</code> objects from the source files, one script for
each dataset. Each script is named as follows: <code>make-data_</code> +
<code>DATASET_NAME</code> + <code>.R</code>.</p>
<p>Note the file called <code>make-metadata.R</code>. It generates a CSV
table required by <code>ExperimentHub</code> where each line corresponds
to a dataset and the columns contains the data source information. The
table is stored in <code>inst/extdata/metadata.csv</code>, which should
never be changed manually.</p>
</div>
<div class="section level3">
<h3 id="r">R/<a class="anchor" aria-label="anchor" href="#r"></a>
</h3>
<p>The folder contains 3 R scripts, but new contributions should only
consider the <code>data.R</code> and can safely ignore the other two.
The <code>data.R</code> script contains the documentation for each
dataset, formatted using <code>roxygen2</code> markup.</p>
</div>
<div class="section level3">
<h3 id="man">man/<a class="anchor" aria-label="anchor" href="#man"></a>
</h3>
<p>The folder contains the compiled documentation manuals, one for each
dataset. These were automatically generated by <code>roxygen2</code> and
should never be changed manually.</p>
</div>
</div>
<div class="section level2">
<h2 id="workflow">Workflow<a class="anchor" aria-label="anchor" href="#workflow"></a>
</h2>
<p>In practice, contributing a new dataset involves 6 steps.</p>
<div class="section level3">
<h3 id="collect-data">1. Collect data<a class="anchor" aria-label="anchor" href="#collect-data"></a>
</h3>
<p>If you want to contribute an already published dataset, identify the
data sources for all feature data and the sample annotations. This is
generally provided in the article, but you may need to request
additional information from the authors.</p>
<p>If you want to contribute with your own dataset, make sure that all
feature data and the sample annotation table are available from a public
repository (eg PRIDE, MASSive or Zenodo).</p>
</div>
<div class="section level3">
<h3 id="create-the-qfeatures-object">2. Create the <code>QFeatures</code> object<a class="anchor" aria-label="anchor" href="#create-the-qfeatures-object"></a>
</h3>
<p>Create a new R script,
<code>inst/scripts/make-data_DATASET_NAME.R</code>, which contains all
the code to convert the data source data into the <code>QFeatures</code>
object. Here are some tips and tricks for generating a high-quality
dataset:</p>
<ul>
<li>Sample annotations are often cluttered, and spread over different
tables or contained within sample names. Generating high-quality sample
annotations may be time-consuming and frustrating. Don’t overlook this
task, sample annotations are essential for rigourous and accurate
downstream analysis.</li>
<li>Converting feature data tables and annotation tables into
<code>QFeatures</code> or <code>SingleCellExperiment</code> objects can
be streamlined using <a href="https://uclouvain-cbio.github.io/scp/reference/readSCP.html" class="external-link"><code>scp::readSCP()</code></a>
and <a href="https://uclouvain-cbio.github.io/scp/reference/readSingleCellExperiment.html" class="external-link"><code>scp::readSingleCellExperiment()</code></a>,
respectively.</li>
<li>Always start with the lowest feature level (eg PSMs). If available,
you should add peptide and protein data using <a href="https://rformassspectrometry.github.io/QFeatures/reference/QFeatures-class.html" class="external-link"><code>QFeatures::addAssay()</code></a>.
You should then add links between the assays. This is streamlined using
<a href="https://rformassspectrometry.github.io/QFeatures/reference/AssayLinks.html" class="external-link"><code>QFeatures::addAssayLink()</code></a>.</li>
<li>Make sure to add data with as little processing as possible. For
instance, MaxQuant provides peptide intensities, but also iBAQ and
MaxLFQ normalised values. You should favour the former over the latter
two, which you could add as supplementary assays (for example, see <a href="https://github.com/UCLouvain-CBIO/scpdata/blob/master/inst/scripts/make-data_woo2022_macrophage.R" class="external-link">here</a>).</li>
</ul>
</div>
<div class="section level3">
<h3 id="document-the-dataset">3. Document the dataset<a class="anchor" aria-label="anchor" href="#document-the-dataset"></a>
</h3>
<p>Add the data documentation and the data collection procedure in
<code>scpdata/R/data.R</code>. Use <code>roxygen2</code> markup
language. The documentation is structured as follows, but you can best
use the documentation of an existing dataset as a template:</p>
<ul>
<li>
<em>Title</em>: First authors et al. Year (Journal): minimal
description.</li>
<li>
<em>Description</em>: short description of the data set. What and
how many cells were acquired? What technology? What is the research
question?</li>
<li>
<em>Format</em>: describe your <code>QFeatures</code> object.
Describe each assay, namely what level features it contains, the number
of features and the number of cells/samples</li>
<li>
<em>Data acquisition</em>: summarise the data acquisition protocol,
namely the sample isolation, sample preparation, liquid chromatography,
mass spectrometry and raw data processing.</li>
<li>
<em>Data collection</em>: summarise the steps you undertook to
generate the <code>QFeatures</code> object, and where to find the script
you created.</li>
<li>
<em>Source</em>: link the public repository with the source
data</li>
<li>
<em>References</em>: if published, refer to the original work that
acquired the data.</li>
<li>
<em>Example</em>: add an example to show how to retrieve the
dataset. To avoid the associated overhead when testing the package, we
recommend adding the example as follows:</li>
</ul>
<pre><code><span><span class="co">##' \donttest{</span></span>
<span><span class="co">##' dataset_name()</span></span>
<span><span class="co">##' }</span></span></code></pre>
<ul>
<li>
<em>Keywords</em>: add the line
<code>##' @keywords datasets</code>
</li>
<li>
<code>"dataset_name"</code>: end the documentation with the name of
your dataset, ensuring your data set is correctly exported.</li>
</ul>
</div>
<div class="section level3">
<h3 id="update-metadata">4. Update metadata<a class="anchor" aria-label="anchor" href="#update-metadata"></a>
</h3>
<p>Add the data source information in the
<code>inst/script/make-metadata.R</code> script and run the complete
script that will update the <code>inst/extdata/metadata.csv</code>. You
can use a previous dataset as template. All fields are mandatory: Title,
Description, BiocVersion, Genome, SourceType, SourceUrl, SourceVersion,
Species, TaxonomyId, Coordinate_1_based, DataProvider, Maintainer,
RDataClass, DispatchClass, PublicationDate, NumberAssays,
PreprocessingSoftware, LabelingProtocol, PsmsAvailable,
PeptidesAvailable, ProteinsAvailable, ContainsSingleCells, Notes. See
<code>?ExperimentHubData::makeExperimentHubMetadata</code> for a
comprehensive description of the fields.</p>
<p>Next, ensure that your updated <code>metadata.csv</code> file is
valid by running
<code>ExperimentHubData::makeExperimentHubMetadata("scpdata")</code>.</p>
</div>
<div class="section level3">
<h3 id="create-a-pull-request">5. Create a pull request<a class="anchor" aria-label="anchor" href="#create-a-pull-request"></a>
</h3>
<p>Push any change you made to GitHub and open a pull request to notify
us of your contribution. The pull request should include all the commits
related to the dataset you want to contribute. Provide in the
description where we can retrieve your <code>QFeatures</code> object,
e.g. through Zenodo.</p>
</div>
<div class="section level3">
<h3 id="almost-done">6. Almost done!<a class="anchor" aria-label="anchor" href="#almost-done"></a>
</h3>
<p>Once your pull request is submitted, we will take over and will
proceed to the following steps:</p>
<ol style="list-style-type: decimal">
<li>We will review your changes to ensure you comply with the above
guidelines. We may eventually request changes.</li>
<li>We will contact the Bioconductor team (<a href="mailto:hubs@bioconductor.org">hubs@bioconductor.org</a>) to upload
your Rda to Microsoft Azure, if needed, and to update the
<code>metadata.csv</code> on their server. See the <a href="https://bioconductor.org/packages/devel/bioc/vignettes/HubPub/inst/doc/CreateAHubPackage.html#uploading-data-to-microsoft-azure-genomic-data-lake" class="external-link">help
page</a> for more information.</li>
<li>We will compile the documentation with roxygen2 and check the
package is still valid. We may eventually request changes.</li>
<li>We will update the NEWS.md file and bump package version</li>
<li>If this is your first contribution, we will add your name to the
package authors.</li>
</ol>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.2.9000.</p>
</div>

      </footer>
</div>






  </body>
</html>
