<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Single Cell Proteomics data sets. • scpdata</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Single Cell Proteomics data sets.">
<meta property="og:description" content="scpdata">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    

    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.1.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Single Cell Proteomics data sets.</h1>
                        <h4 data-toc-skip class="author">Laurent Gatto</h4>
                                    <h4 data-toc-skip class="author">Christophe Vanderaa</h4>
                        
            <h4 data-toc-skip class="date">14 October 2021</h4>
      
      
      <div class="hidden name"><code>scpdata.Rmd</code></div>

    </div>

    
    
<div id="the-scpdata-package" class="section level1">
<h1 class="hasAnchor">
<a href="#the-scpdata-package" class="anchor" aria-hidden="true"></a>The <code>scpdata</code> package</h1>
<p><code>scpdata</code> disseminates mass spectrometry (MS)-based single-cell proteomics (SCP) data sets formatted using the <code>scp</code> data structure. The data structure is described in the <a href="http://www.bioconductor.org/packages/release/bioc/vignettes/scp/inst/doc/scp.html" class="external-link"><code>scp</code> vignette</a>.</p>
<p>In this vignette, we describe how to access the SCP data sets. To start, we load the <code>scpdata</code> package.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st">"scpdata"</span><span class="op">)</span></code></pre></div>
</div>
<div id="load-data-from-experimenthub" class="section level1">
<h1 class="hasAnchor">
<a href="#load-data-from-experimenthub" class="anchor" aria-hidden="true"></a>Load data from <code>ExperimentHub</code>
</h1>
<p>The data is stored using the <a href="https://bioconductor.org/packages/release/bioc/html/ExperimentHub.html" class="external-link"><code>ExperimentHub</code></a> infrastructure. We first create a connection with <code>ExperimentHub</code>.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">eh</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ExperimentHub/man/ExperimentHub-class.html" class="external-link">ExperimentHub</a></span><span class="op">(</span><span class="op">)</span></code></pre></div>
<p>You can list all data sets available in <code>scpdata</code> using the query function.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">query</span><span class="op">(</span><span class="va">eh</span>, <span class="st">"scpdata"</span><span class="op">)</span>
<span class="co">#&gt; ExperimentHub with 13 records</span>
<span class="co">#&gt; # snapshotDate(): 2021-10-13</span>
<span class="co">#&gt; # $dataprovider: PRIDE, MassIVE, SlavovLab website</span>
<span class="co">#&gt; # $species: Homo sapiens, Mus musculus, Rattus norvegicus, Gallus gallus</span>
<span class="co">#&gt; # $rdataclass: QFeatures</span>
<span class="co">#&gt; # additional mcols(): taxonomyid, genome, description,</span>
<span class="co">#&gt; #   coordinate_1_based, maintainer, rdatadateadded, preparerclass, tags,</span>
<span class="co">#&gt; #   rdatapath, sourceurl, sourcetype </span>
<span class="co">#&gt; # retrieve records with, e.g., 'object[["EH3899"]]' </span>
<span class="co">#&gt; </span>
<span class="co">#&gt;            title           </span>
<span class="co">#&gt;   EH3899 | specht2019v2    </span>
<span class="co">#&gt;   EH3900 | specht2019v3    </span>
<span class="co">#&gt;   EH3901 | dou2019_lysates </span>
<span class="co">#&gt;   EH3902 | dou2019_mouse   </span>
<span class="co">#&gt;   EH3903 | dou2019_boosting</span>
<span class="co">#&gt;   ...      ...             </span>
<span class="co">#&gt;   EH3907 | zhu2018NC_islets</span>
<span class="co">#&gt;   EH3908 | cong2020AC      </span>
<span class="co">#&gt;   EH3909 | zhu2019EL       </span>
<span class="co">#&gt;   EH6011 | liang2020_hela  </span>
<span class="co">#&gt;   EH7085 | schoof2021</span></code></pre></div>
<p>Another way to get information about the available data sets is to call <code><a href="../reference/scpdata.html">scpdata()</a></code>. This will retrieve all the available metadata. For example, we can retrieve the data set titles along with the description to make an informed choice about which data set to choose.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">info</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/scpdata.html">scpdata</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; snapshotDate(): 2021-10-13</span>
<span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html" class="external-link">kable</a></span><span class="op">(</span><span class="va">info</span><span class="op">[</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"title"</span>, <span class="st">"description"</span><span class="op">)</span><span class="op">]</span><span class="op">)</span></code></pre></div>
<table class="table">
<colgroup>
<col width="4%">
<col width="12%">
<col width="82%">
</colgroup>
<thead><tr class="header">
<th align="left"></th>
<th align="left">title</th>
<th align="left">description</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">EH3899</td>
<td align="left">specht2019v2</td>
<td align="left">SCP expression data for monocytes (U-937) and macrophages at PSM, peptide and protein level</td>
</tr>
<tr class="even">
<td align="left">EH3900</td>
<td align="left">specht2019v3</td>
<td align="left">SCP expression data for more monocytes (U-937) and macrophages at PSM, peptide and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH3901</td>
<td align="left">dou2019_lysates</td>
<td align="left">SCP expression data for Hela digests (0.2 or 10 ng) at PSM and protein level</td>
</tr>
<tr class="even">
<td align="left">EH3902</td>
<td align="left">dou2019_mouse</td>
<td align="left">SCP expression data for C10, SVEC or Raw cells at PSM and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH3903</td>
<td align="left">dou2019_boosting</td>
<td align="left">SCP expression data for C10, SVEC or Raw cells and 3 boosters (0, 5 or 50 ng) at PSM and protein level</td>
</tr>
<tr class="even">
<td align="left">EH3904</td>
<td align="left">zhu2018MCP</td>
<td align="left">Near SCP expression data for micro-dissection rat brain samples (50, 100, or 200 µm width) at PSM level</td>
</tr>
<tr class="odd">
<td align="left">EH3905</td>
<td align="left">zhu2018NC_hela</td>
<td align="left">Near SCP expression data for HeLa samples (aproximately 12, 40, or 140 cells) at PSM level</td>
</tr>
<tr class="even">
<td align="left">EH3906</td>
<td align="left">zhu2018NC_lysates</td>
<td align="left">Near SCP expression data for HeLa lysates (10, 40 and 140 cell equivalent) at PSM level</td>
</tr>
<tr class="odd">
<td align="left">EH3907</td>
<td align="left">zhu2018NC_islets</td>
<td align="left">Near SCP expression data for micro-dissected human pancreas samples (control patients or type 1 diabetes) at PSM level</td>
</tr>
<tr class="even">
<td align="left">EH3908</td>
<td align="left">cong2020AC</td>
<td align="left">SCP expression data for Hela cells at PSM, peptide and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH3909</td>
<td align="left">zhu2019EL</td>
<td align="left">SCP expression data for chicken utricle samples (1, 3, 5 or 20 cells) at PSM, peptide and protein level</td>
</tr>
<tr class="even">
<td align="left">EH6011</td>
<td align="left">liang2020_hela</td>
<td align="left">Expression data for HeLa cells (0, 1, 10, 150, 500 cells) at PSM, peptide and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH7085</td>
<td align="left">schoof2021</td>
<td align="left">Single-cell proteomics data from OCI-AML8227 cell culture to reconstruct the cellular hierarchy.</td>
</tr>
</tbody>
</table>
<p>To get one of the data sets (<em>e.g.</em> <code>dou2019_lysates</code>) you can either retrieve it using the <code>ExperimentHub</code> query function</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">scp</span> <span class="op">&lt;-</span> <span class="va">eh</span><span class="op">[[</span><span class="st">"EH3901"</span><span class="op">]</span><span class="op">]</span>
<span class="co">#&gt; see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="va">scp</span>
<span class="co">#&gt; An instance of class QFeatures containing 3 assays:</span>
<span class="co">#&gt;  [1] Hela_run_1: SingleCellExperiment with 35224 rows and 10 columns </span>
<span class="co">#&gt;  [2] Hela_run_2: SingleCellExperiment with 35080 rows and 10 columns </span>
<span class="co">#&gt;  [3] proteins: SingleCellExperiment with 1641 rows and 20 columns</span></code></pre></div>
<p>or you can the use the built-in functions from <code>scpdata</code></p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">scp</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/dou2019_lysates.html">dou2019_lysates</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; see ?scpdata and browseVignettes('scpdata') for documentation</span>
<span class="co">#&gt; loading from cache</span>
<span class="va">scp</span>
<span class="co">#&gt; An instance of class QFeatures containing 3 assays:</span>
<span class="co">#&gt;  [1] Hela_run_1: SingleCellExperiment with 35224 rows and 10 columns </span>
<span class="co">#&gt;  [2] Hela_run_2: SingleCellExperiment with 35080 rows and 10 columns </span>
<span class="co">#&gt;  [3] proteins: SingleCellExperiment with 1641 rows and 20 columns</span></code></pre></div>
</div>
<div id="data-sets-information" class="section level1">
<h1 class="hasAnchor">
<a href="#data-sets-information" class="anchor" aria-hidden="true"></a>Data sets information</h1>
<p>Each data set has been extensively documented in a separate man page (<em>e.g.</em> <code><a href="../reference/dou2019_lysates.html">?dou2019_lysates</a></code>). You can find information about the data content, the acquisition protocol, the data collection procedure as well as the data sources and reference.</p>
</div>
<div id="data-manipulation" class="section level1">
<h1 class="hasAnchor">
<a href="#data-manipulation" class="anchor" aria-hidden="true"></a>Data manipulation</h1>
<p>For more information about manipulating the data sets, check the <a href="http://bioconductor.org/packages/release/bioc/html/scp.html" class="external-link"><code>scp</code></a> package. The <code>scp</code> <a href="http://bioconductor.org/packages/release/bioc/vignettes/scp/inst/doc/scp.html" class="external-link">vignette</a> will guide you through a typical SCP data processing workflow. Once your data is loaded from <code>scpdata</code> you can skip section 2 <em>Read in SCP data</em> of the <code>scp</code> vignette.</p>
</div>
<div id="session-information" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#session-information" class="anchor" aria-hidden="true"></a>Session information</h1>
<pre><code>R version 4.1.1 (2021-08-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.3 LTS

Matrix products: default
BLAS/LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.8.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=C             
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] scpdata_1.1.2                ExperimentHub_2.1.4         
 [3] AnnotationHub_3.1.5          BiocFileCache_2.1.1         
 [5] dbplyr_2.1.1                 QFeatures_1.3.6             
 [7] MultiAssayExperiment_1.19.16 SummarizedExperiment_1.23.5 
 [9] Biobase_2.53.0               GenomicRanges_1.45.0        
[11] GenomeInfoDb_1.29.8          IRanges_2.27.2              
[13] S4Vectors_0.31.5             BiocGenerics_0.39.2         
[15] MatrixGenerics_1.5.4         matrixStats_0.61.0          
[17] BiocStyle_2.21.4            

loaded via a namespace (and not attached):
 [1] ProtGenerics_1.25.1           bitops_1.0-7                 
 [3] fs_1.5.0                      bit64_4.0.5                  
 [5] filelock_1.0.2                httr_1.4.2                   
 [7] rprojroot_2.0.2               tools_4.1.1                  
 [9] bslib_0.3.1                   utf8_1.2.2                   
[11] R6_2.5.1                      DBI_1.1.1                    
[13] lazyeval_0.2.2                withr_2.4.2                  
[15] tidyselect_1.1.1              bit_4.0.4                    
[17] curl_4.3.2                    compiler_4.1.1               
[19] textshaping_0.3.6             cli_3.0.1                    
[21] desc_1.4.0                    DelayedArray_0.19.4          
[23] bookdown_0.24                 sass_0.4.0                   
[25] rappdirs_0.3.3                pkgdown_1.9000.9000.9000     
[27] systemfonts_1.0.3             stringr_1.4.0                
[29] digest_0.6.28                 rmarkdown_2.11               
[31] XVector_0.33.0                pkgconfig_2.0.3              
[33] htmltools_0.5.2               highr_0.9                    
[35] fastmap_1.1.0                 rlang_0.99.0.9000            
[37] RSQLite_2.2.8                 shiny_1.7.1                  
[39] jquerylib_0.1.4               generics_0.1.0               
[41] jsonlite_1.7.2                dplyr_1.0.7                  
[43] RCurl_1.98-1.5                magrittr_2.0.1               
[45] GenomeInfoDbData_1.2.7        Matrix_1.3-4                 
[47] Rcpp_1.0.7                    fansi_0.5.0                  
[49] MsCoreUtils_1.5.0             lifecycle_1.0.1              
[51] stringi_1.7.5                 yaml_2.2.1                   
[53] MASS_7.3-54                   zlibbioc_1.39.0              
[55] grid_4.1.1                    blob_1.2.2                   
[57] promises_1.2.0.1              crayon_1.4.1                 
[59] lattice_0.20-45               Biostrings_2.61.2            
[61] KEGGREST_1.33.0               knitr_1.36                   
[63] pillar_1.6.3                  igraph_1.2.6                 
[65] glue_1.4.2                    BiocVersion_3.14.0           
[67] evaluate_0.14                 BiocManager_1.30.16          
[69] png_0.1-7                     vctrs_0.3.8                  
[71] httpuv_1.6.3                  purrr_0.3.4                  
[73] clue_0.3-60                   assertthat_0.2.1             
[75] cachem_1.0.6                  xfun_0.26                    
[77] mime_0.12                     xtable_1.8-4                 
[79] AnnotationFilter_1.17.1       later_1.3.0                  
[81] ragg_1.1.3                    SingleCellExperiment_1.15.2  
[83] tibble_3.1.5                  AnnotationDbi_1.55.1         
[85] memoise_2.0.0                 cluster_2.1.2                
[87] ellipsis_0.3.2                interactiveDisplayBase_1.31.2</code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link external-link">pkgdown</a> 1.9000.9000.9000.</p>
</div>

      </footer>
</div>

  


  

  </body>
</html>
