<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Single Cell Proteomics data sets â€¢ scpdata</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Single Cell Proteomics data sets">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">scpdata</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.17.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/scpdata.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/contribution_guidelines.html">Contribution guidelines</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Single Cell Proteomics data sets</h1>
                        <h4 data-toc-skip class="author">Laurent
Gatto</h4>
                                    <h4 data-toc-skip class="author">Christophe
Vanderaa</h4>
                        
            <h4 data-toc-skip class="date">19 May 2025</h4>
      

      <div class="hidden name"><code>scpdata.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="the-scpdata-package">The <code>scpdata</code> package<a class="anchor" aria-label="anchor" href="#the-scpdata-package"></a>
</h2>
<p><code>scpdata</code> disseminates mass spectrometry (MS)-based
single-cell proteomics (SCP) data sets formatted using the
<code>scp</code> data structure. The data structure is described in the
<a href="http://www.bioconductor.org/packages/release/bioc/vignettes/scp/inst/doc/scp.html" class="external-link"><code>scp</code>
vignette</a>.</p>
<p>In this vignette, we describe how to access the SCP data sets. To
start, we load the <code>scpdata</code> package.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st">"scpdata"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="load-data-from-experimenthub">Load data from <code>ExperimentHub</code><a class="anchor" aria-label="anchor" href="#load-data-from-experimenthub"></a>
</h2>
<p>The data is stored using the <a href="https://bioconductor.org/packages/release/bioc/html/ExperimentHub.html" class="external-link"><code>ExperimentHub</code></a>
infrastructure. We first create a connection with
<code>ExperimentHub</code>.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">eh</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/ExperimentHub/man/ExperimentHub-class.html" class="external-link">ExperimentHub</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<p>You can list all data sets available in <code>scpdata</code> using
the query function.</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">query</span><span class="op">(</span><span class="va">eh</span>, <span class="st">"scpdata"</span><span class="op">)</span></span>
<span><span class="co">#&gt; ExperimentHub with 30 records</span></span>
<span><span class="co">#&gt; # snapshotDate(): 2025-04-21</span></span>
<span><span class="co">#&gt; # $dataprovider: MassIVE, PRIDE, SlavovLab website, Dataverse</span></span>
<span><span class="co">#&gt; # $species: Homo sapiens, Mus musculus, Rattus norvegicus, Gallus gallus</span></span>
<span><span class="co">#&gt; # $rdataclass: QFeatures, SingleCellExperiment</span></span>
<span><span class="co">#&gt; # additional mcols(): taxonomyid, genome, description,</span></span>
<span><span class="co">#&gt; #   coordinate_1_based, maintainer, rdatadateadded, preparerclass, tags,</span></span>
<span><span class="co">#&gt; #   rdatapath, sourceurl, sourcetype </span></span>
<span><span class="co">#&gt; # retrieve records with, e.g., 'object[["EH3899"]]' </span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;            title                  </span></span>
<span><span class="co">#&gt;   EH3899 | specht2019v2           </span></span>
<span><span class="co">#&gt;   EH3900 | specht2019v3           </span></span>
<span><span class="co">#&gt;   EH3901 | dou2019_lysates        </span></span>
<span><span class="co">#&gt;   EH3902 | dou2019_mouse          </span></span>
<span><span class="co">#&gt;   EH3903 | dou2019_boosting       </span></span>
<span><span class="co">#&gt;   ...      ...                    </span></span>
<span><span class="co">#&gt;   EH9498 | petrosius2023_AstralAML</span></span>
<span><span class="co">#&gt;   EH9609 | krull2024              </span></span>
<span><span class="co">#&gt;   EH9610 | hu2023_K562            </span></span>
<span><span class="co">#&gt;   EH9611 | hu2023_oocyte          </span></span>
<span><span class="co">#&gt;   EH9627 | ai2025a</span></span></code></pre></div>
<p>Another way to get information about the available data sets is to
call <code><a href="../reference/scpdata.html">scpdata()</a></code>. This will retrieve all the available
metadata. For example, we can retrieve the data set titles along with
the description to make an informed choice about which data set to
choose.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">info</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/scpdata.html">scpdata</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html" class="external-link">kable</a></span><span class="op">(</span><span class="va">info</span><span class="op">[</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"title"</span>, <span class="st">"description"</span><span class="op">)</span><span class="op">]</span><span class="op">)</span></span></code></pre></div>
<table class="table">
<colgroup>
<col width="4%">
<col width="16%">
<col width="79%">
</colgroup>
<thead><tr class="header">
<th align="left"></th>
<th align="left">title</th>
<th align="left">description</th>
</tr></thead>
<tbody>
<tr class="odd">
<td align="left">EH3899</td>
<td align="left">specht2019v2</td>
<td align="left">SCP expression data for monocytes (U-937) and
macrophages at PSM, peptide and protein level</td>
</tr>
<tr class="even">
<td align="left">EH3900</td>
<td align="left">specht2019v3</td>
<td align="left">SCP expression data for more monocytes (U-937) and
macrophages at PSM, peptide and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH3901</td>
<td align="left">dou2019_lysates</td>
<td align="left">SCP expression data for Hela digests (0.2 or 10 ng) at
PSM and protein level</td>
</tr>
<tr class="even">
<td align="left">EH3902</td>
<td align="left">dou2019_mouse</td>
<td align="left">SCP expression data for C10, SVEC or Raw cells at PSM
and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH3903</td>
<td align="left">dou2019_boosting</td>
<td align="left">SCP expression data for C10, SVEC or Raw cells and 3
boosters (0, 5 or 50 ng) at PSM and protein level</td>
</tr>
<tr class="even">
<td align="left">EH3904</td>
<td align="left">zhu2018MCP</td>
<td align="left">Near SCP expression data for micro-dissection rat brain
samples (50, 100, or 200 Âµm width) at PSM level</td>
</tr>
<tr class="odd">
<td align="left">EH3905</td>
<td align="left">zhu2018NC_hela</td>
<td align="left">Near SCP expression data for HeLa samples (aproximately
12, 40, or 140 cells) at PSM level</td>
</tr>
<tr class="even">
<td align="left">EH3906</td>
<td align="left">zhu2018NC_lysates</td>
<td align="left">Near SCP expression data for HeLa lysates (10, 40 and
140 cell equivalent) at PSM level</td>
</tr>
<tr class="odd">
<td align="left">EH3907</td>
<td align="left">zhu2018NC_islets</td>
<td align="left">Near SCP expression data for micro-dissected human
pancreas samples (control patients or type 1 diabetes) at PSM level</td>
</tr>
<tr class="even">
<td align="left">EH3908</td>
<td align="left">cong2020AC</td>
<td align="left">SCP expression data for Hela cells at PSM, peptide and
protein level</td>
</tr>
<tr class="odd">
<td align="left">EH3909</td>
<td align="left">zhu2019EL</td>
<td align="left">SCP expression data for chicken utricle samples (1, 3,
5 or 20 cells) at PSM, peptide and protein level</td>
</tr>
<tr class="even">
<td align="left">EH6011</td>
<td align="left">liang2020_hela</td>
<td align="left">Expression data for HeLa cells (0, 1, 10, 150, 500
cells) at PSM, peptide and protein level</td>
</tr>
<tr class="odd">
<td align="left">EH7085</td>
<td align="left">schoof2021</td>
<td align="left">Single-cell proteomics data from OCI-AML8227 cell
culture to reconstruct the cellular hierarchy.</td>
</tr>
<tr class="even">
<td align="left">EH7295</td>
<td align="left">williams2020_lfq</td>
<td align="left">Single-cell label free proteomics data from a MCF10A
cell line culture.</td>
</tr>
<tr class="odd">
<td align="left">EH7296</td>
<td align="left">williams2020_tmt</td>
<td align="left">Single-cell proteomics data from three acute myeloid
leukemia cell line culture (MOLM-14, K562, CMK).</td>
</tr>
<tr class="even">
<td align="left">EH7712</td>
<td align="left">derks2022</td>
<td align="left">Single-cell and bulk (100-cell) proteomics data of
PDAC, melanoma cells and monocytes.</td>
</tr>
<tr class="odd">
<td align="left">EH7713</td>
<td align="left">brunner2022</td>
<td align="left">Single-cell proteomics data of cell cycle stages in
HeLa.</td>
</tr>
<tr class="even">
<td align="left">EH8301</td>
<td align="left">leduc2022_pSCoPE</td>
<td align="left">Single-cell proteomics data of 878 melanoma cells and
877 monocytes (pSCoPE).</td>
</tr>
<tr class="odd">
<td align="left">EH8302</td>
<td align="left">leduc2022_plexDIA</td>
<td align="left">Single-cell proteomics data of 126 melanoma cells
(plexDIA).</td>
</tr>
<tr class="even">
<td align="left">EH8303</td>
<td align="left">woo2022_macrophage</td>
<td align="left">Single-cell proteomics data from LPS-treated
macrophages.</td>
</tr>
<tr class="odd">
<td align="left">EH8304</td>
<td align="left">woo2022_lung</td>
<td align="left">Single-cell proteomics data from primary human lung
cells.</td>
</tr>
<tr class="even">
<td align="left">EH9450</td>
<td align="left">gregoire2023_mixCTRL</td>
<td align="left">Single-cell proteomics data from two monocyte cell
lines</td>
</tr>
<tr class="odd">
<td align="left">EH9477</td>
<td align="left">khan2023</td>
<td align="left">Single-cell proteomics data of 421 MCF-10A cells
undergoing EMT triggered by TGF-beta</td>
</tr>
<tr class="even">
<td align="left">EH9487</td>
<td align="left">guise2024</td>
<td align="left">Single-cell proteomics data of 108 postmortem CTL or
ALS spinal moto neurons</td>
</tr>
<tr class="odd">
<td align="left">EH9497</td>
<td align="left">petrosius2023_mES</td>
<td align="left">Mouse embryonic stem cells across ground-state (m2i)
and differentiation-permissive (m15) culture conditions.</td>
</tr>
<tr class="even">
<td align="left">EH9498</td>
<td align="left">petrosius2023_AstralAML</td>
<td align="left">Single-cell proteomics data of 4 cell types from the
OCI-AML8227 model.</td>
</tr>
<tr class="odd">
<td align="left">EH9609</td>
<td align="left">krull2024</td>
<td align="left">Single-cell proteomics data IFN-Y response of U-2 OS
cells</td>
</tr>
<tr class="even">
<td align="left">EH9610</td>
<td align="left">hu2023_K562</td>
<td align="left">Single-cell proteomics data of K562 cells</td>
</tr>
<tr class="odd">
<td align="left">EH9611</td>
<td align="left">hu2023_oocyte</td>
<td align="left">Single-cell proteomics data of oocytes</td>
</tr>
<tr class="even">
<td align="left">EH9627</td>
<td align="left">ai2025a</td>
<td align="left">Single-cell proteomics data of adult cardiomyocytes
from Ai et al.Â (2025)</td>
</tr>
</tbody>
</table>
<p>To get one of the data sets (<em>e.g.</em>
<code>dou2019_lysates</code>) you can either retrieve it using the
<code>ExperimentHub</code> query function</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">scp</span> <span class="op">&lt;-</span> <span class="va">eh</span><span class="op">[[</span><span class="st">"EH3901"</span><span class="op">]</span><span class="op">]</span></span>
<span><span class="co">#&gt; see ?scpdata and browseVignettes('scpdata') for documentation</span></span>
<span><span class="co">#&gt; loading from cache</span></span>
<span><span class="va">scp</span></span>
<span><span class="co">#&gt; An instance of class QFeatures containing 4 set(s):</span></span>
<span><span class="co">#&gt;  [1] Hela_run_1: SingleCellExperiment with 24562 rows and 10 columns </span></span>
<span><span class="co">#&gt;  [2] Hela_run_2: SingleCellExperiment with 24310 rows and 10 columns </span></span>
<span><span class="co">#&gt;  [3] peptides: SingleCellExperiment with 13934 rows and 20 columns </span></span>
<span><span class="co">#&gt;  [4] proteins: SingleCellExperiment with 1641 rows and 20 columns</span></span></code></pre></div>
<p>or you can the use the built-in functions from
<code>scpdata</code></p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">scp</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/dou2019_lysates.html">dou2019_lysates</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; see ?scpdata and browseVignettes('scpdata') for documentation</span></span>
<span><span class="co">#&gt; loading from cache</span></span>
<span><span class="va">scp</span></span>
<span><span class="co">#&gt; An instance of class QFeatures containing 4 set(s):</span></span>
<span><span class="co">#&gt;  [1] Hela_run_1: SingleCellExperiment with 24562 rows and 10 columns </span></span>
<span><span class="co">#&gt;  [2] Hela_run_2: SingleCellExperiment with 24310 rows and 10 columns </span></span>
<span><span class="co">#&gt;  [3] peptides: SingleCellExperiment with 13934 rows and 20 columns </span></span>
<span><span class="co">#&gt;  [4] proteins: SingleCellExperiment with 1641 rows and 20 columns</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="data-sets-information">Data sets information<a class="anchor" aria-label="anchor" href="#data-sets-information"></a>
</h2>
<p>Each data set has been extensively documented in a separate man page
(<em>e.g.</em> <code><a href="../reference/dou2019_lysates.html">?dou2019_lysates</a></code>). You can find information
about the data content, the acquisition protocol, the data collection
procedure as well as the data sources and reference.</p>
</div>
<div class="section level2">
<h2 id="data-manipulation">Data manipulation<a class="anchor" aria-label="anchor" href="#data-manipulation"></a>
</h2>
<p>For more information about manipulating the data sets, check the <a href="http://bioconductor.org/packages/release/bioc/html/scp.html" class="external-link"><code>scp</code></a>
package. The <code>scp</code> <a href="http://bioconductor.org/packages/release/bioc/vignettes/scp/inst/doc/scp.html" class="external-link">vignette</a>
will guide you through a typical SCP data processing workflow. Once your
data is loaded from <code>scpdata</code> you can skip section 2 <em>Read
in SCP data</em> of the <code>scp</code> vignette.</p>
</div>
<div class="section level2">
<h2 class="unnumbered" id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<pre><code>R version 4.5.0 (2025-04-11)
Platform: x86_64-pc-linux-gnu
Running under: Ubuntu 24.04.2 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.26.so;  LAPACK version 3.12.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

time zone: UTC
tzcode source: system (glibc)

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] scpdata_1.17.1              ExperimentHub_2.99.0       
 [3] AnnotationHub_3.99.1        BiocFileCache_2.99.4       
 [5] dbplyr_2.5.0                QFeatures_1.19.0           
 [7] MultiAssayExperiment_1.35.3 SummarizedExperiment_1.39.0
 [9] Biobase_2.69.0              GenomicRanges_1.61.0       
[11] GenomeInfoDb_1.45.3         IRanges_2.43.0             
[13] S4Vectors_0.47.0            BiocGenerics_0.55.0        
[15] generics_0.1.4              MatrixGenerics_1.21.0      
[17] matrixStats_1.5.0           BiocStyle_2.37.0           

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.1            dplyr_1.1.4                
 [3] blob_1.2.4                  Biostrings_2.77.1          
 [5] filelock_1.0.3              SingleCellExperiment_1.31.0
 [7] fastmap_1.2.0               lazyeval_0.2.2             
 [9] digest_0.6.37               lifecycle_1.0.4            
[11] cluster_2.1.8.1             ProtGenerics_1.41.0        
[13] KEGGREST_1.49.0             RSQLite_2.3.11             
[15] magrittr_2.0.3              compiler_4.5.0             
[17] rlang_1.1.6                 sass_0.4.10                
[19] tools_4.5.0                 igraph_2.1.4               
[21] yaml_2.3.10                 knitr_1.50                 
[23] S4Arrays_1.9.0              htmlwidgets_1.6.4          
[25] bit_4.6.0                   curl_6.2.2                 
[27] DelayedArray_0.35.1         plyr_1.8.9                 
[29] abind_1.4-8                 withr_3.0.2                
[31] purrr_1.0.4                 desc_1.4.3                 
[33] grid_4.5.0                  MASS_7.3-65                
[35] cli_3.6.5                   rmarkdown_2.29             
[37] crayon_1.5.3                ragg_1.4.0                 
[39] httr_1.4.7                  reshape2_1.4.4             
[41] BiocBaseUtils_1.11.0        DBI_1.2.3                  
[43] cachem_1.1.0                stringr_1.5.1              
[45] AnnotationDbi_1.71.0        AnnotationFilter_1.33.0    
[47] BiocManager_1.30.25         XVector_0.49.0             
[49] vctrs_0.6.5                 Matrix_1.7-3               
[51] jsonlite_2.0.0              bookdown_0.43              
[53] bit64_4.6.0-1               clue_0.3-66                
[55] systemfonts_1.2.3           tidyr_1.3.1                
[57] jquerylib_0.1.4             glue_1.8.0                 
[59] pkgdown_2.1.3.9000          stringi_1.8.7              
[61] BiocVersion_3.22.0          UCSC.utils_1.5.0           
[63] tibble_3.2.1                pillar_1.10.2              
[65] rappdirs_0.3.3              htmltools_0.5.8.1          
[67] R6_2.6.1                    httr2_1.1.2                
[69] textshaping_1.0.1           evaluate_1.0.3             
[71] lattice_0.22-7              png_0.1-8                  
[73] memoise_2.0.1               bslib_0.9.0                
[75] Rcpp_1.0.14                 SparseArray_1.9.0          
[77] xfun_0.52                   MsCoreUtils_1.21.0         
[79] fs_1.6.6                    pkgconfig_2.0.3            </code></pre>
</div>
<div class="section level2">
<h2 id="license">License<a class="anchor" aria-label="anchor" href="#license"></a>
</h2>
<p>This vignette is distributed under a <a href="https://creativecommons.org/licenses/by-sa/2.0/" class="external-link">CC BY-SA
license</a>.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Christophe Vanderaa, Laurent Gatto.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.3.9000.</p>
</div>

      </footer>
</div>






  </body>
</html>
